src/inventory/ModelManager.cpp

```diff
@@ -82,9 +84,24 @@ ModelManager::ModelManager(QObject *parent, Settings * settings)
             QDir().mkpath(configDir_.absolutePath());
         }
     }
+    
+    // Update RepoManager with data from settings (also reload on update of setting)
+    repositories_.load(settings_->externalRepos.value());
```

Comment:
We should probably notify the user that once a new repo is added, they need to explicitly click "Download new models".

Reply:
It's implicitly communicated right now. The code just below that bit clears `remoteModels_`, the list will only show local ones and the _Download new models_ option after changing repositories:
```
        // We do clear the remoteModels list so that it is clear that it is
        // outdated and/or incomplete. Now users can click the "download model
        // list" again and make an informed decision to access the internet.
        remoteModels_.clear();
        updateAvailableModels();
```
A popup saying "hey click that button" every time repositories are updated is probably annoying. And/or going to be ignored. Nobody ever reads popups…

Reply:
Or to be more clear about it: they have _no other option_ than to click that "download new models" option if they want to download any model at all after updating the repositories. The flow is exactly like they're already used to at that point.

Reply:
Ok, fair point.